code/game/objects/items/control_wand.dm

```diff
@@ -138,7 +138,7 @@
 	desc = "A device used for illegally interfacing with doors."
 	icon_state = "hacktool"
 	item_state = "hacktool"
-	var/hack_speed = 30
+	var/hack_speed = 15
```

Comment:
```suggestion
	var/hack_speed = 1.5 SECONDS
```
Also your pr cl suggests it's been doubled, like to 6 seconds, might want to rephrase

Reply:
Oh yeah that's just a conflict of saying it easily for the non-coders and the var name, will clarify